easybuild/easyblocks/generic/rpackage.py

```diff
@@ -195,9 +174,20 @@ def install_R_package(self, cmd, inp=None):
         else:
             self.log.debug("R package %s installed succesfully" % self.name)
 
+    def update_config_guess(self, path):
+        """Update any config.guess found in specified directory"""
+        for config_guess_dir in (root for root, _, files in os.walk(path) if 'config.guess' in files):
+            config_guess = os.path.join(config_guess_dir, 'config.guess')
+            if not check_config_guess(config_guess):
+                updated_config_guess = obtain_config_guess(log=self.log)
+                copy_file(updated_config_guess, config_guess)
+                self.log.debug("Replacing outdated %s with more recent copy at %s", config_guess, updated_config_guess)
```

Comment:
Either make it `Replaced`, or move up this `log.debug` call (and maybe make it `log.info`)?